documentation/modules/snip-migrating-luns.adoc

```diff
@@ -0,0 +1,10 @@
+:_content-type: SNIPPET
+
+[NOTE]
+====
+{project-short} supports direct migration of logical units numbers (LUNs) for {rhv-full} ({rhv-short}) source providers as follows:
+
+* LUNs are detached from the source provider and then attached to the target providers, rather than being copied directly.
```

Comment:
the `rather than being copied directly` sounds a bit bad to me. it's true compared to other disks, but i don't think LUNs are expected to be copied at all as the type of storage is direct LUN. but, i can't think of something else to highlight it.

Reply:
@liranr23 Would it better to say " LUNs are detached from the source provider and then attached to the target providers." and leave it at that?

Reply:
yeah, i think it's better.